diff --git a/config_perms.module b/config_perms.module
index a7a7603..0351fc5 100755
--- a/config_perms.module
+++ b/config_perms.module
@@ -15,10 +15,11 @@
  * Load all permissions
  */
 function config_perms_perms($machine_name = NULL, $rebuild = FALSE) {
-  // Load current perms
-  $perms = cache_get('config_perms');
+  // Load current perms.
+  $cache_obj = cache_get('config_perms');
+  $perms = (is_object($cache_obj) && isset($cache_obj->data)) ? $cache_obj->data : false;
 
-  // Rebuild if not there
+  // Rebuild if not there.
   if (!$perms || $rebuild) {
     $perms = config_perms_cache_rebuild();
   }
@@ -32,7 +33,7 @@ function config_perms_perms($machine_name = NULL, $rebuild = FALSE) {
 function config_perms_cache_rebuild() {
   $perms = config_perms_loadperms();
   cache_set('config_perms', $perms);
-  
+
   return $perms;
 }
 
@@ -102,11 +103,11 @@ function config_perms_menu() {
 /**
  * Implements hook_menu_alter().
  */
-function config_perms_menu_alter(&$items) {  
+function config_perms_menu_alter(&$items) {
   // Run through all menu items and leter them accordingly
   foreach($items as $key => $item) {
     foreach (config_perms_perms() as $perm) {
-      foreach ($perm->path as $path) { 
+      foreach ($perm->path as $path) {
         if (drupal_match_path($key, $path) && $perm->status) {
           $items[$key]['access callback'] = 'config_perms_access_callback';
           $items[$key]['access arguments'] = array($perm);
@@ -120,6 +121,9 @@ function config_perms_menu_alter(&$items) {
  * Access callback
  */
 function config_perms_access_callback($perm) {
+  if(is_string($perm)) {
+    return user_access(check_plain($perm));
+  }
   return user_access(check_plain($perm->name));
 }
 
@@ -170,7 +174,7 @@ function config_perms_save($perm) {
   else {
     drupal_write_record('config_perms', $perm);
   }
-  
+
   return $perm;
 }
 
