diff --git a/context_ui/context_ui.module b/context_ui/context_ui.module
index ea5003d..d97c78e 100644
--- a/context_ui/context_ui.module
+++ b/context_ui/context_ui.module
@@ -366,16 +366,19 @@ function context_ui_page_alter(&$page) {
  * to activate it.
  */
 function context_ui_menu_contextual_links_alter(&$links, $router_item, $root_path) {
-  if(context_ui_dialog_is_enabled()  &&
-     !context_isset('context_ui', 'context_ui_editor_present')) {
-    $links['layout'] = array(
-      'href' => 'context-ui/activate',
-      'title' => t('Configure Layout'),
-      'localized_options' => array(
-        'query' =>  array('destination'=> $_GET['q']),
-        'options' => array('html' => FALSE, 'attributes' => array()),
-      ),
-    );
+  global $user;
+  if (in_array('developer', array_values($user->roles)) || in_array('support', array_values($user->roles))) {
+    if(context_ui_dialog_is_enabled()  &&
+      !context_isset('context_ui', 'context_ui_editor_present')) {
+      $links['layout'] = array(
+        'href' => 'context-ui/activate',
+        'title' => t('Configure Layout'),
+        'localized_options' => array(
+          'query' =>  array('destination'=> $_GET['q']),
+          'options' => array('html' => FALSE, 'attributes' => array()),
+        ),
+      );
+    }
   }
 }
 
