diff --git a/seotools.info b/seotools.info
index ba70751..9a55f6c 100644
--- a/seotools.info
+++ b/seotools.info
@@ -1,7 +1,7 @@
-name = "SEO Tools"
+name = "SEO Tools Extension"
 description = "Suite of advanced search engine optimization tools for Drupal"
 core = "7.x"
-package = "SEO"
+package = "Pro 101 Infrastructure (G)"
 php = "5.2.4"
 dependencies[] = "alchemy"
 dependencies[] = "alchemy_contentanalysis"
@@ -25,4 +25,4 @@ features[ctools][] = "strongarm:strongarm:1"
 features[variable][] = "insight_contentanalysis_autorun"
 features[variable][] = "insight_contentanalysis_autorun_analyzers"
 apps[server] = 'development'
-apps[manifests][] = 'app/manifest.app'
\ No newline at end of file
+apps[manifests][] = 'app/manifest.app'
