diff --git a/views/workbench_current_user.view.inc b/views/workbench_current_user.view.inc
index ff9cd7e..2ae8cfd 100644
--- a/views/workbench_current_user.view.inc
+++ b/views/workbench_current_user.view.inc
@@ -27,6 +27,15 @@ $handler->display->display_options['pager']['type'] = 'none';
 $handler->display->display_options['pager']['options']['offset'] = '0';
 $handler->display->display_options['style_plugin'] = 'default';
 $handler->display->display_options['row_plugin'] = 'fields';
+/* Footer: Locked Documents */
+$handler->display->display_options['footer']['area']['id'] = 'area';
+$handler->display->display_options['footer']['area']['table'] = 'views';
+$handler->display->display_options['footer']['area']['field'] = 'area';
+$handler->display->display_options['footer']['area']['ui_name'] = 'Locked Documents';
+$handler->display->display_options['footer']['area']['empty'] = TRUE;
+$handler->display->display_options['footer']['area']['content'] = '<a href="user/[field_first_name]-[field_last_name]/content_lock">Locked Documents</a>';
+$handler->display->display_options['footer']['area']['format'] = 'plain_text';
+$handler->display->display_options['footer']['area']['tokenize'] = 1;
 /* Field: User: Uid */
 $handler->display->display_options['fields']['uid']['id'] = 'uid';
 $handler->display->display_options['fields']['uid']['table'] = 'users';
